iroh/src/magicsock/relay_actor.rs

```diff
@@ -600,19 +592,21 @@ impl RelayActor {
             }
         }
         let url = found_relay.as_ref().unwrap_or(url);
-        self.active_relay_handle(url).await
+        self.active_relay_handle(url)
     }
 
     /// Returns the handle of the [`ActiveRelayActor`].
-    async fn active_relay_handle(&mut self, url: &RelayUrl) -> &ActiveRelayHandle {
+    fn active_relay_handle(&mut self, url: &RelayUrl) -> &ActiveRelayHandle {
         if !self.active_relays.contains_key(url) {
             let handle = self.start_active_relay(url.clone());
             if Some(url) == self.msock.my_relay().as_ref() {
-                handle
+                if let Err(err) = handle
                     .inbox_addr
-                    .send(ActiveRelayMessage::SetHomeRelay(true))
-                    .await
-                    .ok();
+                    .try_send(ActiveRelayMessage::SetHomeRelay(true))
+                {
+                    error!("Send to newly started active relay failed: {err:#}.");
+                    warn!("Home relay not set");
```

Comment:
why to log messages?

Reply:
Not sure...
- One is to explain what the bug is.
- The other explains what the impact is.

I was very close to calling `.expect()` on this to let it panic.  I think it probably is a bug if you can't send to the inbox right after creating it (and it was nicer to have this method not be async).  But I went with very noisy logging instead.